bidict/_loose.py

```diff
@@ -0,0 +1,9 @@
+from ._bidict import bidict
+
+class loosebidict(bidict):
+    """
+    A mutable bidict which always uses forcing put operations
+    so that it never raises :class:`ValueExistsException`.
+    """
+    def _put(self, key, val):
```

Comment:
You're right. We shouldn't make changes like this unless we've profiled what the possible speedups might be.